src/CMakeLists.txt

```diff
@@ -18,8 +22,23 @@ target_link_libraries(kdenow
     KF5::Service
 )
 
+target_link_libraries(kdenowd
+    kdenowcore
+)
+
+generate_export_header(kdenowcore
+    EXPORT_FILE_NAME kdenowcore_export.h
+    EXPORT_MACRO_NAME KDENOWCORE_EXPORT
+)
+
 set_target_properties(
```

Comment:
We have to export only kdenowcore, not kdenowd. So that other projects can use it.